Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CallbackMapProperties.java

```diff
@@ -91,6 +91,16 @@ public String getUsername() {
      * @see gov.hhs.fha.nhinc.callback.openSAML.CallbackProperties#getAuthenicationContextClass()
      */
     @Override
+    public DateTime getIssueInstant() {
```

Comment:
There was an issueInstant property defined for the Evidence Build that was used in for the Conditions element for AuthzDecisionStatement.